src/main/java/io/vertx/core/json/JsonArray.java

```diff
@@ -92,6 +92,20 @@ public JsonArray(Buffer buf) {
     }
   }
 
+  /**
+   * Create a JsonArray containing an arbitrary number of values.
+   *
+   * @param values The objects into JsonArray.
+   * @throws NullPointerException if the args is null.
+   */
+  public static JsonArray of(Object... values) {
+    // implicit nullcheck of args
+    if (args.length == 0) {
+      return new JsonArray();
+    }
+    return new JsonArray(new ArrayList<>(Arrays.asList(values)));
```

Comment:
we should avoid Arrays.asList() as it creates an intermediary list that is not necessary, instead we should add the values to the created JsonArray like we do for JsonObject

Reply:
Ok, fixed @vietj